### PR TITLE
Add section on quarto computatations

### PR DESCRIPTION
**Pull request by mahesh-panchal**
A short intro on computations in Quarto.

### CODE REVIEW

File: quarto/intro.qmd [hunk 1]
```diff
@@ -8,7 +8,77 @@
...
+title: My Experiment
+execute:
+  echo: false
+---
+````
+
+Alternatively, execution options can be specified within each code block, e.g.,
+
+````
+```{python}
```
Comment (gbdias) [inline quarto/intro.qmd:68]: I'm not sure how you want this displayed, but on my side for the code block to show up I need to double curly brace the language declaration {{python}}

**Review by gbdias — COMMENTED** [review]
Looks good, I just added a few comments about syntax escaping but I admit I'm not sure how some of these blocks are supposed to be rendered in the html document.

File: quarto/intro.qmd [hunk 2]
```diff
@@ -8,7 +8,77 @@
...
 ### Render vs preview (Guilherme / Estelle)
 
-## Computations (Mahesh / Guilherme)
+## Computations
+
+Quarto lets you perform computations within your notebook. This is typically
+done using code blocks denoted by three backticks followed by the language 
+you're using in curly brackets.
+
+```
```
[the remark this answers is not in the document]
Comment (mahesh-panchal) [reply quarto/intro.qmd]: Thanks. It seems I misread the documentation.

File: quarto/intro.qmd (hunk 1 above)
Comment (mahesh-panchal) [reply quarto/intro.qmd]: I could have sworn the documentation had it like this, but I can't find it now. 
Anyway, checked it all with `quarto preview` and it renders now how I want.

**Review by gbdias — APPROVED** [review]

File: quarto/intro.qmd [hunk 3]
```diff
@@ -8,7 +8,76 @@
...
+Additional languages can also be supported through other Jupyter kernels (see Engines below).
+See this page for a list of [Jupyter kernels](https://gist.github.com/chronitis/682c4e0d9f663e85e3d87e97cd7d1624).
+
+The languages and packages used in your computations must be available in your `render` environment, and 
+are often installed through other means, for example using `conda` or a container platform.
+
+Packages and variables created within one code block are also accessible from other code blocks, including
+inline code blocks. For example, here we create a figure and reference the value in the caption.
+
+````{.markdown}
```
Comment (percyfal) [inline quarto/intro.qmd:39]: Even though the section is not about references per se, could be nice to include the markdown code to actually reference a figure? E.g., "see @fig-plot-alt for a plot of $y=x^2$.

**Review by percyfal — COMMENTED** [review]
LGTM, possibly add fig reference.

File: quarto/intro.qmd (hunk 3 above)
Comment (mahesh-panchal) [reply quarto/intro.qmd:40]: Updated text and made it more about the referencing.

**Review by percyfal — APPROVED** [review]

**Pull request merged**